apps/regattatimer/regattatimer.translations.json

```diff
@@ -0,0 +1,10 @@
+{
+  "de": {
+    "speed": "FüG",
+    "speed_unit": "kn"
+  },
+  "en": {
+    "speed": "SOA",
+    "speed_unit": "kn"
+  }
+}
```

Comment:
Could probably have been done with /*LANG*/ and the translation added to the corresponding file in lang/

Reply:
Correct. Tbh, the presence of `/*LANG*/` in the code does not help with readability. But mainly I did this because I thought you should not push very specific stuff like this to the main lang repo. But I can do that in a later release.